tests/workbenches/utils.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright Skodjob authors.
+# License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+#
+from __future__ import annotations
+
+import pathlib
+from contextlib import contextmanager
+from typing import Generator
+
+import yaml
+from pytest_testconfig import config as py_config
+
+from kubernetes.dynamic import DynamicClient, Resource, ResourceInstance
+
+from ocp_resources.route import Route
+from ocp_resources.notebook import Notebook
+
+from utilities.constants import INTERNAL_IMAGE_REGISTRY_PATH
+
+
+def get_notebook_image(image_name: str, image_tag: str) -> str:
+    controllers_namespace = py_config["applications_namespace"]
+    if py_config.get("distribution") == "upstream":
+        image_dict = {"jupyter-minimal-notebook": "jupyter-minimal-notebook"}
+    else:
+        image_dict = {"jupyter-minimal-notebook": "s2i-minimal-notebook"}
+    return f"{INTERNAL_IMAGE_REGISTRY_PATH}/{controllers_namespace}/{image_dict[image_name]}:{image_tag}"
+
+
+def load_default_notebook(dyn_client: DynamicClient, namespace: str, name: str, image: str) -> Notebook:
+    notebook_string = (pathlib.Path(__file__).parent / "notebook-controller/test_data/notebook.yaml").read_text()
```

Comment:
I think it's better to load the YAML here and work with dict which is python object and not with strings.

Reply:
ack, will see if I can get approvals from rhoai people without making the change; personally I don't want to make it

Reply:
please work with a dict; it is more readable and pythonic

Reply:
done